### PR TITLE
Docs improvement v3

### PR DESCRIPTION
**Pull request by heyjustinai**
# What does this PR do?

- updated the notebooks to reflect past changes up to llama-stack 0.0.53
- updated readme to  provide accurate and up-to-date info
- improve the current zero to hero by integrating an example using together api

## Before submitting

- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Ran pre-commit to handle lint / formatting issues.
- [x] Read the [contributor guideline](https://github.com/meta-llama/llama-stack/blob/main/CONTRIBUTING.md),
      Pull Request section?
- [ ] Updated relevant documentation.
- [ ] Wrote necessary unit or integration tests.

### CODE REVIEW

**Comment by ashwinb** [comment]
Is this not rebased on origin/main? I am a little confused. I see some of the changes we made recently to docs represented in here.

**Comment by heyjustinai** [comment]
> Is this not rebased on origin/main? I am a little confused. I see some of the changes we made recently to docs represented in here.

oops! that's my b, let me correct that

**Comment by heyjustinai** [comment]
> Is this not rebased on origin/main? I am a little confused. I see some of the changes we made recently to docs represented in here.

oops! that's my b, let me correct that

**Comment by heyjustinai** [comment]
fixed! @ashwinb

File: zero_to_hero_guide/quickstart.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
+# Quickstart Guide
```
Comment (ashwinb) [inline zero_to_hero_guide/quickstart.md:1]: let's move this under docs/ please and let's add a prominent pointer to the README. Even at the top-level breadcrumbs right now would be fine with me.

File: zero_to_hero_guide/quickstart.md [hunk 2]
```diff
@@ -82,20 +102,35 @@ If you're looking for more specific topics like tool calling or agent setup, we
...
+export INFERENCE_MODEL="meta-llama/Llama-3.2-3B-Instruct"
+export SAFETY_MODEL="meta-llama/Llama-Guard-3-1B"
+```
 
 3. **Run the Llama Stack**:
-   - Run the stack with the configured YAML file:
+   - Run the stack with command shared by the API from earlier:
      ```bash
-     llama stack run /path/to/your/distro/llamastack-ollama/ollama-run.yaml --port 5050
+     llama stack run /Users/username/.llama/distributions/llamastack-ollama/ollama-run.yaml  \
```
Comment (ashwinb) [inline zero_to_hero_guide/quickstart.md:124]: you can simply do `llama stack run ollama` and that would be sufficient I think

Comment (heyjustinai) [reply zero_to_hero_guide/quickstart.md]: thanks! made all the necessary changes

**Review by ashwinb — APPROVED** [review]

**Pull request merged**